docker-compose.yml

```diff
@@ -26,34 +31,43 @@ services:
       DATABASE_URL: postgresql://postgres:password@postgres:5432/flagsmith
       USE_POSTGRES_FOR_ANALYTICS: 'true' # Store API and Flag Analytics data in Postgres
 
-      ENV: prod # set to "prod" in production.
+      ENV: prod # set to 'prod' in production.
       DJANGO_ALLOWED_HOSTS: '*' # Change this in production
       ALLOW_ADMIN_INITIATION_VIA_CLI: 'true' # Change this in production
-      FLAGSMITH_DOMAIN: 'localhost:8000' # Change this in production
-      DJANGO_SECRET_KEY: 'secret' # Change this in production
-      # PREVENT_SIGNUP: "true" # Uncomment to prevent additional signups
-      # ENABLE_ADMIN_ACCESS_USER_PASS: "true" # set to "true" to enable access to the /admin/ Django backend via your username and password
-      # ALLOW_REGISTRATION_WITHOUT_INVITE: "true"
+      FLAGSMITH_DOMAIN: localhost:8000 # Change this in production
+      DJANGO_SECRET_KEY: secret # Change this in production
+      # PREVENT_SIGNUP: 'true' # Uncomment to prevent additional signups
+      # ENABLE_ADMIN_ACCESS_USER_PASS: 'true' # Uncomment to enable access to the /admin/ Django backend via your username and password
+      # ALLOW_REGISTRATION_WITHOUT_INVITE: 'true'
 
       # Enable Task Processor
       TASK_RUN_METHOD: TASK_PROCESSOR # other options are: SYNCHRONOUSLY, SEPARATE_THREAD (default)
+
       # For more info on configuring E-Mails - https://docs.flagsmith.com/deployment/locally-api#environment-variables
-      #
       # Example SMTP:
       # EMAIL_BACKEND: django.core.mail.backends.smtp.EmailBackend
       # EMAIL_HOST: mail.example.com
       # SENDER_EMAIL: flagsmith@example.com
       # EMAIL_HOST_USER: flagsmith@example.com
       # EMAIL_HOST_PASSWORD: smtp_account_password
       # EMAIL_PORT: 587 # optional
-      # EMAIL_USE_TLS: "true" # optional
-
+      # EMAIL_USE_TLS: 'true' # optional
+    healthcheck:
+      test:
+        [
+          'CMD-SHELL',
+          'python -c "url = ''http://localhost:8000/health'';import requests;import sys;status =
+          requests.get(url).status_code;sys.exit(0 if 200 >= status < 300 else 1)"',
+        ]
```

Comment:
Can we extract this to a script in api/scripts perhaps? Then this could become `python ./scripts/healthcheck.py` for example and the script itself can have some more whitespace so it's easier to understand.